core/src/main/java/org/apache/carbondata/core/scan/executor/util/RestructureUtil.java

```diff
@@ -355,12 +338,11 @@ public static Object getMeasureDefaultValueByType(ColumnSchema columnSchema,
    * @param blockExecutionInfo
    * @param queryMeasures        measures present in query
    * @param currentBlockMeasures current block measures
-   * @param isUnManagedTable
    * @return measures present in the block
    */
   public static List<ProjectionMeasure> createMeasureInfoAndGetCurrentBlockQueryMeasures(
       BlockExecutionInfo blockExecutionInfo, List<ProjectionMeasure> queryMeasures,
-      List<CarbonMeasure> currentBlockMeasures, boolean isUnManagedTable) {
+      List<CarbonMeasure> currentBlockMeasures) {
```

Comment:
For non transactional table columnid check should not be persent during read. As files can be created some where else and can be copied.